Code/source/Parser.h

```diff
@@ -1,3 +1,4 @@
 #pragma once
 using namespace std;
-int Parse();
\ No newline at end of file
+
+int Parse(std::string simpleCode);
```

Comment:
dont a header need 
class nameOfClass {
public:
           constructor.....
           functions..........
}

Reply:
I will change this later subsequently when we finalise what is suppose to be inside Parser.cpp